src/plugins/cgal/nodes/normals_generator.cpp

```diff
@@ -26,7 +29,7 @@ struct FakeKernel {
 		Vector_3(const std::array<float, 3>& arr) : data(arr) {
 		}
 
-		Vector_3(CGAL::Null_vector) : data{{0, 0, 0}} {
+		Vector_3(const CGAL::Null_vector) : data{{0, 0, 0}} {
```

Comment:
Codacy found an issue: [Struct 'Vector_3' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/martin.prazak/possumwood/pullRequest?prid=6280524)